.changelog/140.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:breaking-change
+The cloud-packer service was updated to grpc_gateway v2, this changes the payloads for the requests in some routes, as they are now embedded within the request rather than referenced from it.
```

Comment:
Might help to include what payload type was changed and to which new type

Reply:
I don't have the full list (and it might be quite extensive tbh), I can probably give a few examples though, would that be alright?